PittAPI/people.py

```diff
@@ -17,61 +17,63 @@
 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 '''
 
-import urllib.parse
-
 import grequests
 from typing import List, Dict, Iterator, Any
 
 PEOPLE_URL = "https://m.pitt.edu/people/search.json"
 DETAIL_URL = "https://m.pitt.edu/people/detail.json"
-
-
-def get_person(query: str, max_people: int = 10) -> List[Dict[str,Any]]:
-    """ """
-    url_list = [item for l_list in _get_person_url(query, max_people) for item in l_list]
-    results = [_get_person_details(url) for url in url_list]
-    people_info = [resp.json()["response"] for resp in grequests.map(results)]
+FIELD_MAP = {
+    "ldap:cn" : "name",
+    "ldap:buildingname": "address",
+    "kgoperson:phone" : "phone",
+    "ldap:ou" : "school",
```

Comment:
Please change this from `school` to `department`. For example, if I look up myself it says my school is CSSD instead of Dietrich. Pretty much the only consistent case of it actually being the school is mostly with faculty.f So just make it department just like how it is on find.pitt.edu.

Reply:
Hmm, sure, good point.